### DISCUSSION TITLE
[Help] Addon Texture Pack

### DISCUSSION THREAD

**Discussion by Gekocaretaker1** [bridge-core/bridge.#39] [Q&A] [answered]
How do I use the texture pack feature in 1.13.4 Bridge?

**Comment by Gekocaretaker1** [comment]
Also is it possible to have 3d items?

**Comment by solvedDev** [comment, marked as answer]
@Gekocaretaker1 There's a dedicated sidebar icon (image) you can click to get into the resource pack manager. If you haven't linked a RP yet, you can choose to link an existing pack or create a new one. 

It is not possible to do 3D items yet and you shouldn't post bridge. unrelated questions to the issue tracker. ;)
